src/batch/src/executor2/join/chunked_data.rs

```diff
@@ -0,0 +1,212 @@
+// Copyright 2022 Singularity Data
```

Comment:
We should also remove `chunked_data` in `executor/join` mod?

Reply:
I tried this once but had some problems, if I delete` executor/join/chunked_data`, many functions and structures under `executor2/join/chunked_data` have to be changed to pub, so I keep `executor/join/chunked_data` for now and delete it after all of them have been ported?